### DISCUSSION TITLE
Multiplayer spectate is turning transparent and maximizing a player only layers them over all other ones.

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22211] [Q&A] [answered]
The video speaks for itself, is this an issue or my fault? It happend right after I left a lobby and then joined back in to spectate, my dim setting is at 100% and everything worked as intended 5 min. prior to the recording within the same session.

https://user-images.githubusercontent.com/87947227/212521333-83134ece-3aab-409f-90d3-e0497ec58940.mp4

https://user-images.githubusercontent.com/87947227/212521335-64d51365-206e-4867-9f92-cb6e1a969fc9.mp4

**Comment by bdach** [comment, marked as answer]
Can you link the map? I believe this issue may be related to the specific beatmap.

**Reply by Inconsist** [reply]
https://osu.ppy.sh/beatmapsets/663519#osu/1418502

its the right diff

**Reply by bdach** [reply]
Yeah, we're aware of this. There's even an (incomplete) fix open for this.

I'll convert this to an issue for tracking though.

**Reply by bdach** [reply]
Tracking at https://github.com/ppy/osu/issues/22238.
